src/main/resources/org/terasology/launcher/bundle/LabelsBundle_de.properties

```diff
@@ -121,6 +121,7 @@ settings_launcher_closeLauncherAfterGameStart=Launcher beim Spielstart schlie\u0
 settings_launcher_launcherDirectory=Launcher Nutzerdatenverzeichnis
 settings_launcher_launcherDirectory_open=\u00D6ffnen
 settings_launcher_saveDownloadedFiles=Heruntergeladene Pakete behalten
+settings_launcher_showPreReleases=Prereleases (Vorabversionen) und Testversionen anzeigen
```

Comment:
```suggestion
settings_launcher_showPreReleases=Pre-Releases (Vorabversionen) und Testversionen anzeigen
```

Reply:
According to https://de.wikipedia.org/wiki/Entwicklungsstadium_(Software) it's "Prerelease" :shrug: What does the Duden say about it?